lib/plugins/aws/deploy/compile/functions/tests/index.js

```diff
@@ -114,12 +114,17 @@ describe('AwsCompileFunctions', () => {
         .IamPolicyLambdaExecution
         .Properties
         .PolicyName
-      ).to.equal(
-        `${
-          awsCompileFunctions.options.stage
-        }-${
-          awsCompileFunctions.serverless.service.service
-        }-lambda`
+      ).to.deep.equal(
+        {
+          'Fn:Join': [
+            '-',
+            [
+              '${opt:stage}',
+              '${self:service}',
```

Comment:
No, the variables are only populated after loading of the service once. So adding more into the CF template doesn't work. What we can do though is putting those into the CF template as variables so they can be referenced.